sdk/storage/azure-storage-blob/src/test/java/com/azure/storage/blob/ContainerAPITest.groovy

```diff
@@ -1172,18 +1173,18 @@ class ContainerAPITest extends APISpec {
         def blobs = cc.listBlobs().iterator()
 
         then:
-        blobs.next().getName() == name
-        blobs.next().getName() == name + "2"
-        blobs.next().getName() == name + "3"
+        Utility.urlDecode(blobs.next().getName()) == name
```

Comment:
See above comment on decoding in getters. We should actually be validating that get without decoding is equal to the name.

Reply:
See above answer about this same issue.